docs/reference/commandline/cli.md

```diff
@@ -265,6 +273,15 @@ Following is a sample `config.json` file:
 {% endraw %}
 ```
 
+### Experimental features
+
+To enable [experimental features](https://success.docker.com/article/experimental-features),
```

Comment:
We don't usually link to success.docker.com from the cli/docs repo because it is enterprise-related documentation. It would, however, be appropriate for success.docker.com to link to the reference docs of cli.